src/shared/src/components/organisms/AltinnReceipt.test.tsx

```diff
@@ -30,14 +30,30 @@ const attachment2 = {
 };
 
 describe('AltinnReceipt', () => {
-  it('should show titleSubmitted when set', () => {
+  it('should not show titleSubmitted when there are no pdfs', () => {
     render();
 
+    expect(
+      screen.queryByRole('heading', {
+        name: /titlesubmitted/i,
+      }),
+    ).not.toBeInTheDocument();
+
+    // No list of attachments expected
```

Comment:
nitpick; this comment is maybe a bit redundant? 🙂

Same on line 55.

They say the same thing as the expect assertion, just with different words

Reply:
I disagree somewhat, but I also expected this comment.. ;-) I think I would find it non-obvious this is meant to look for the attachment list, as there could be any amount of lists in the receipt component (there is already a table, which is in itself a kind of listing). I looked at the query documentation in order to construct a more specific query, but looking for the specific dom-id was apparently advised against. I'd prefer one comment too many rather than having to look through history for the original intention. I might represent a minority on this though.

Reply:
I would also find it non-obvious that this comment is talking about a specific list (even though "attachments" is mentioned). A list is a list🙂 Its not easy to understand that underlying info just by reading this (which is what happened to me, and so I wrote the initial comment😅 ). So I would argue that it is still redundant - the comment and the expect has the same meaning for someone who doesnt know the context.

There is a "safeguard" here though in the `queryByRole` function. If there are more than 1 returned items, it will throw an error. This will not cover the case where the `list` is replaced with another incorrect list though (f.ex which contains something other than PDF's).

We could alternatively change this to `queryByTestId` f.ex, and attach a more specific `data-testid` attribute to the element we are looking for. Then we can still change the implementation without updating the test (but would have to remember to keep the data attribute)